README.adoc

```diff
@@ -247,3 +247,9 @@ server:
 |===
 
 == Additional Notes
+
+=== Running the UI outside of the core
+When developing the UI is useful to run it outside of the core to make it easier
```

Comment:
missing some words here between "UI" and "is". Though, it is hard explaining the English as for why that is :). But, I think it should be:

`When developing the UI you will find it useful...`

It could also simply be, "When developing the UI it is useful to run it ...." however the double use of the word "it" sounds weird.

Reply:
Updated with the suggestion